test/controllers/drivers_controller_test.rb

```diff
@@ -2,174 +2,379 @@
 
 describe DriversController do
   # Note: If any of these tests have names that conflict with either the requirements or your team's decisions, feel empowered to change the test names. For example, if a given test name says "responds with 404" but your team's decision is to respond with redirect, please change the test name.
+  before do
+    @driver = Driver.create(name: "Driver 123", vin: "WBWSS52P9NEYLVDE9", available: true, isactive: true)
+  end
 
   describe "index" do
     it "responds with success when there are many drivers saved" do
       # Arrange
       # Ensure that there is at least one Driver saved
+      driver = @driver
 
       # Act
-
+      get "/drivers"
+      
       # Assert
-
+      expect(Driver.count).must_be :>=, 1
+      must_respond_with :success
     end
 
     it "responds with success when there are no drivers saved" do
       # Arrange
+      Driver.delete_all
+      
       # Ensure that there are zero drivers saved
-
       # Act
+      get "/drivers"
 
       # Assert
-
+      expect(Driver.count).must_equal 0
+      must_respond_with :success
     end
   end
 
   describe "show" do
     it "responds with success when showing an existing valid driver" do
       # Arrange
       # Ensure that there is a driver saved
+      valid_driver_id = @driver.id
 
       # Act
+      get "/drivers/#{ valid_driver_id }"
 
       # Assert
-
+      must_respond_with :success
     end
 
     it "responds with 404 with an invalid driver id" do
       # Arrange
       # Ensure that there is an id that points to no driver
+      invalid_driver_id = -1
 
-      # Act
-
+      # Act 
+      get "/drivers/#{ invalid_driver_id }"
+      
       # Assert
+      must_respond_with :not_found
+    end
+
+    it "responds with 404 with an inactive driver id" do
+      # Arrange
+      # Ensure that there is an id that points to inactive driver
+      @driver.update(available: false, isactive: false)
+      inactive_driver_id = @driver.id
 
+      # Act 
+      get "/drivers/#{ inactive_driver_id }"
+      
+      # Assert
+      must_respond_with :not_found
     end
   end
 
   describe "new" do
+    it "can get the new_driver_path" do
+      get new_driver_path
+
+      must_respond_with :success
+    end
+
     it "responds with success" do
+      get new_driver_path(Driver.first.id)
+
+      must_respond_with :success
     end
```

Comment:
I don't understand why this is here.  You don't create a new driver with another driver's id.